ui/src/components/table/QTable.json

```diff
@@ -788,7 +788,17 @@
         "cols": {
           "type": "Object",
```

Comment:
I believe this should be an Array. This would mean the end type should be `{ name: string, value: any, ... }[]`.